### PR TITLE
Enhanced conversion 1.1

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
Cleanup of unused features of previous version + simple check based on ToS acceptance and CTA click.

### CODE REVIEW

File: _dev/apps/ui/src/components/enhanced-conversions/alert-sign-gads-tos.vue [hunk 1]
```diff
@@ -32,18 +35,22 @@
...
   computed: {
-    ...mapGetters('app', [
-      GettersTypesApp.GET_FEATURE_FLAG_ENHANCED_CONVERSIONS,
-    ]),
     linkToTermsOfServices(): string {
       return googleUrl.googleAdsConversionsTermsAndConditions;
     },
+    linkToInstructions(): string {
+      // TODO: Replace with actual URL
+      return 'perdu.com';
```
Comment (ga-devfront) [inline _dev/apps/ui/src/components/enhanced-conversions/alert-sign-gads-tos.vue:47]: Should you replace it or not ?

Comment (Quetzacoalt91) [reply _dev/apps/ui/src/components/enhanced-conversions/alert-sign-gads-tos.vue:47]: Of course, I'm waiting for the actual URL

File: _dev/apps/ui/src/components/enhanced-conversions/enhanced-conversions-card.vue [hunk 2]
```diff
@@ -57,40 +51,30 @@
...
-    };
-  },
   computed: {
-    ...mapGetters('app', [
-      GettersTypesApp.GET_FEATURE_FLAG_ENHANCED_CONVERSIONS,
-    ]),
     ...mapGetters('campaigns', [
       GettersTypesCampaigns.GET_ENHANCED_CONVERSIONS_STATUS,
+      GettersTypesCampaigns.GET_ENHANCED_CONVERSIONS_TOGGABLE,
     ]),
```
Comment (ga-devfront) [inline _dev/apps/ui/src/components/enhanced-conversions/enhanced-conversions-card.vue:67]: you can use mapGetters as an object to not reuse this computed to create another one =>

```suggestion
    ...mapGetters('campaigns', {
      GET_ENHANCED_CONVERSIONS_STATUS: GettersTypesCampaigns.GET_ENHANCED_CONVERSIONS_STATUS,
      enhancedConversionIsToggable: GettersTypesCampaigns.GET_ENHANCED_CONVERSIONS_TOGGABLE,
    }),
```
see documentation => https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper

**Review by ga-devfront — APPROVED** [review]
Everything seems ok I just added a comment for a double computed of the same variable because of vueX.
image

File: _dev/apps/ui/src/components/enhanced-conversions/enhanced-conversions-card.vue (hunk 2 above)
Comment (Quetzacoalt91) [reply _dev/apps/ui/src/components/enhanced-conversions/enhanced-conversions-card.vue:67]: I removed the computed that was useless

**Review by ga-devfront — APPROVED** [review]

**Pull request merged**